src/staticOptions.ts

```diff
@@ -187,6 +206,18 @@ const staticOptions = asLanguageClientOptionsById({
     ],
     mutualizable: true
   },
+  postgresql: {
+    documentSelector: [{
+      language: 'sql'
```

Comment:
I think we do have `postgresql` language, don't we?

Reply:
we do: https://github.com/CodinGame/monaco-editor-wrapper/blob/8dab3a81e72306118d950b166c437d74262d2365/src/languages/extensions/languages.json#L1126

Reply:
rebased 6cac88ccb9a4071c5534ff8d665154377b41f527